Batchbald_implementation.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    savefig("accuracy")
</pre><p>Can you give a better explicit name to figure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Changed it to test_accuracy_curve. Does it work?